### PR TITLE
Added community drivers

### PR DESCRIPTION
**Pull request by stefanak-michal**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-bolt-63.surge.sh

Updated pages:
- [Neo4j Drivers](https://neo4j-docs-bolt-63.surge.sh/bolt/1.0/neo4j-drivers)

**Review by robsdedude — APPROVED** [review]
:sunglasses:

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-bolt-63.surge.sh

Updated pages:
- [Neo4j Drivers](https://neo4j-docs-bolt-63.surge.sh/bolt/1.0/neo4j-drivers)

**Comment by fbiville** [comment]
There are a few more still:

- Haskell: https://github.com/zmactep/hasbolt
- C#: https://github.com/DotNet4Neo4j/Neo4jClient
- C: https://github.com/majensen/libneo4j-omni

**Review by stefano-ottolenghi — APPROVED** [review]

**Comment by robsdedude** [comment]
@fbiville 

> * Haskell: https://github.com/zmactep/hasbolt
Doesn't seem to support bolt 4. It's soon going to stop being relevant and 4.4 goes EOL.

> * C#: https://github.com/DotNet4Neo4j/Neo4jClient
This is an OGM, if I understand right. I wouldn't call these drivers. As far as I can tell, it's wrapping the official .NET driver.

> * C: https://github.com/majensen/libneo4j-omni
Good catch, I'll add it.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-bolt-63.surge.sh

Updated pages:
- [Neo4j Drivers](https://neo4j-docs-bolt-63.surge.sh/bolt/1.0/neo4j-drivers)

File: modules/ROOT/pages/neo4j-drivers.adoc [hunk 1]
```diff
@@ -16,14 +16,16 @@
...
+
+** Perl: link:https://github.com/majensen/perlbolt[Neo4j::Bolt]
 
 ** PHP: link:https://github.com/neo4j-php/[neo4j-php]
 
+** Ruby: link:https://github.com/neo4jrb[neo4j.rb]
+
 ** Rust: link:https://github.com/neo4j-labs/neo4rs[neo4rs]
 
 ** Perl: link:https://github.com/johannessen/neo4j-driver-perl[Neo4j::Driver]
```
Comment (johannessen) [inline modules/ROOT/pages/neo4j-drivers.adoc:31]: If you include the Perl modules here, they should probably be listed next to each other.

(Nice idea to add the community drivers, BTW.)

**Review by johannessen — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/neo4j-drivers.adoc (hunk 1 above)
Comment (robsdedude) [reply modules/ROOT/pages/neo4j-drivers.adoc:31]: Thanks. I messed that one up :sweat_smile:

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-bolt-63.surge.sh

Updated pages:
- [Neo4j Drivers](https://neo4j-docs-bolt-63.surge.sh/bolt/1.0/neo4j-drivers)

**Review by johannessen — APPROVED** [review]

**Pull request merged**